src/Domain/Imagine/Filter/FilterExecutor.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Domain\Imagine\Filter;
+
+use Imagine\Image\ImageInterface;
+
+interface FilterExecutor
```

Comment:
better name for Liip\ImagineBundle\Imagine\Filter\Loader\LoaderInterface - functionality does not need to be changed.